miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/SamplePage.java

```diff
@@ -0,0 +1,501 @@
+    if ("input".equals(tubeNumberLabel.getTagName())) {
+      return tubeNumberLabel.getAttribute("value");
+    } else {
+      return tubeNumberLabel.getText();
+    }
+  }
+
+  public void setTubeNumber(String tubeNumber) {
+    setText(tubeNumber, tubeNumberLabel);
+  }
+
+  public String getTissueMaterial() {
+    return getSelectedDropdownText(tissueMaterialLabel);
+  }
+
+  public void setTissueMaterial(String tissueMaterial) {
+    setDropdown(tissueMaterial, tissueMaterialLabel);
+  }
+
+  public String getRegion() {
+    return regionLabel.getAttribute("value");
+  }
+
+  public void setRegion(String region) {
+    setText(region, regionLabel);
+  }
+
+  public String getExternalInstituteIdentifier() {
+    return externalInstituteIdentifierLabel.getAttribute("value");
+  }
+
+  public void setExternalInstituteIdentifier(String externalInstituteIdentifier) {
+    setText(externalInstituteIdentifier, externalInstituteIdentifierLabel);
+  }
+
+  public String getLab() {
+    return getSelectedDropdownText(labLabel);
+  }
+
+  public void setLab(String lab) {
+    setDropdown(lab, labLabel);
+  }
+
+  public String getSlidesRemaining() {
+    return slidesRemainingLabel.getText();
+  }
+
+  public void setSlidesRemaining(String slidesRemaining) {
+    setText(slidesRemaining, slidesRemainingLabel);
+  }
+
+  public String getSlides() {
+    return slidesLabel.getAttribute("value");
+  }
+
+  public void setSlides(String slides) {
+    setText(slides, slidesLabel);
+  }
+
+  public String getSlideDiscards() {
+    return discardsLabel.getAttribute("value");
+  }
+
+  public void setSlideDiscards(String discards) {
+    setText(discards, discardsLabel);
+  }
+
+  public String getSlideThickness() {
+    return thicknessLabel.getAttribute("value");
+  }
+
+  public void setSlideThickness(String thickness) {
+    setText(thickness, thicknessLabel);
+  }
+
+  public String getSlideStain() {
+    return getSelectedDropdownText(stainLabel);
+  }
+
+  public void setSlideStain(String stain) {
+    setDropdown(stain, stainLabel);
+  }
+
+  public String getSlidesConsumed() {
+    return slidesConsumedLabel.getAttribute("value");
+  }
+
+  public void setSlidesConsumed(String slidesConsumed) {
+    setText(slidesConsumed, slidesConsumedLabel);
+  }
+
+  public String getStrStatus() {
+    return getSelectedDropdownText(strStatusLabel).trim();
+  }
+
+  public void setStrStatus(String strStatus) {
+    setDropdown(strStatus, strStatusLabel);
+  }
+
+  public String getDnaseTreated() {
+    return dnaseTreatedLabel.getAttribute("value");
+  }
+
+  public void setDnaseTreated(Boolean dnaseTreated) {
+    setCheckbox(dnaseTreated, dnaseTreatedLabel);
+  }
+
+  public String getSamplePurpose() {
+    return getSelectedDropdownText(samplePurposeLabel);
+  }
+
+  public void setSamplePurpose(String samplePurpose) {
+    setDropdown(samplePurpose, samplePurposeLabel);
+  }
+
+  public String getSensibleDate(String date) {
```

Comment:
The Edit Sample page is the only place where this particular date format (dd/MM/yyyy) is used. The Library page uses dd-mmm-yyyy and the Handsontable uses yyyy-MM-dd.